src/components/Datatable/BatchModule/ElementCounter/ElementCounter.tsx

```diff
@@ -58,7 +58,11 @@ const ElementCounter: React.FC<ElementCounterProps> = ({
   );
 
   return (
-    <H3 margin="none" variant={HeadingVariants.secondary}>
+    <H3
+      margin="none"
+      padding={{ horizontal: 0.4 }}
```

Comment:
this should be done in https://github.com/securityscorecard/design-system/blob/datatable/src/components/Datatable/BatchModule/BatchModule.tsx#L20

Reply:
Right. moving it there